backend/logic/users/users.js

```diff
@@ -181,20 +205,117 @@ var getStudentData = async studentID => {
 
   let data;
   let conn = await mysql.getNewConnection();
-  let courses, major, minor;
+  let completedCourses, incompletedCourses, major, minors;
   try {
-    courses = await conn.query(
+    completedCourses = await conn.query(
       `SELECT course_code, semester
     FROM course_offerings
     WHERE (id, semester)
     IN (SELECT offering_id, semester FROM student_course_offerings WHERE student_id = ?);`,
       [studentID]
     );
-    major = await conn.query(`SELECT curriculum_name FROM student_majors WHERE student_id = ?;`, [studentID]);
-    minor = await conn.query(`SELECT curriculum_name FROM student_minors WHERE student_id = ?;`, [studentID]);
+    major = await conn.query(
+      `SELECT curriculum_name FROM student_majors WHERE student_id = ?;`,
+      [studentID]
+    );
+    minors = await conn.query(
+      `SELECT curriculum_name FROM student_minors WHERE student_id = ?;`,
+      [studentID]
+    );
+
+    let currYear = new Date().getFullYear();
+    let currMonth = new Date().getMonth();
+    let fallSem, winterSem;
+    let curriculumName = major[0].curriculum_name;
+
+    if (currMonth < 3) {
```

Comment:
What about Summer?

Reply:
Current webscraping doesn't include summer courses so it should be fine but I agree that it should be supported. I suggest adding a low priority task for refactoring this code to include summer terms and minors later on :)